src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/JavaScriptEncoder.cs

```diff
@@ -73,6 +73,9 @@ public DefaultJavaScriptEncoder(TextEncoderSettings filter)
 
             _allowedCharacters.ForbidCharacter('\\');
             _allowedCharacters.ForbidCharacter('/');
+            
+            // Disallow GRAVE ACCEN \u0060 character as it is supported in ECMAScript v6
```

Comment:
Thanks for the catch. I'll fix Accent and will remove the part saying supported. also I want to mention Unicode standard is defining this character as GRAVE ACCEN and not back-tick string-quoting. this is why I used the standard defined name instead.